projects/1.18/assets/create-cafe/createcafe/lang/zh_cn.json

```
@@ -1 +1,171 @@
-{}
\ No newline at end of file
+{
+    "block.createcafe.melted_sugar": "融化的糖",
+    "block.createcafe.filling": "填充",

```

```suggestion
    "block.createcafe.filling": "奥利奥夹心",
```
游戏内是填入奥利奥的流体，这是它唯一的用途。

---

扭一扭